src/lib/y2firewall/importer_strategies/suse_firewall.rb

```diff
@@ -0,0 +1,214 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "y2firewall/firewalld"
+
+module Y2Firewall
+  module ImporterStrategies
+    # This class is reponsible of parsing SuSEFirewall2 firewalld profile's
+    # section configuring the Y2Firewall::Firewalld instance according to it.
+    class SuseFirewall
+      # [Hash] AutoYaST profile firewall's section
+      attr_accessor :profile
+
+      # SuSEFirewall2 zones
+      ZONES = ["DMZ", "INT", "EXT"].freeze
+
+      # @return [Array<string>] list of zones
+      def zones
+        ZONES
+      end
+
+      # Constructor
+      #
+      # @param [Hash] AutoYaST profile firewall's section
+      def initialize(profile)
+        @profile = profile
+      end
+
+      # It process the profile configuring the firewalld zones that match
+      # better with the SuSEFirewalld ones.
```

Comment:
SuSEFirewalld?

Reply:
xDDD too many names .. I meant SuSEFirewall2 fixing it.